src/EnergyPlus/DXCoils.cc

```diff
@@ -8456,7 +8458,7 @@ Label50: ;
 			}
 			EIR = DXCoil( DXCoilNum ).RatedEIR( Mode ) * EIRTempModFac * EIRFlowModFac;
 			// Calculate modified PartLoadRatio due to defrost (reverse-cycle defrost only)
-			PLRHeating = min( 1.0, ( PartLoadRatio + LoadDueToDefrost / TotCap ) );
+			PLRHeating = min( 1.0, ( PartLoadRatio + LoadDueToDefrost / TotCapAdj ) );
 			if ( DXCoil( DXCoilNum ).DXCoilType_Num != CoilVRF_Heating ) {
```

Comment:
This PLR thing is a great question without a clear answer. It's not really part of the bug I'm fixing here, though. The reason this line changed is because I changed a variable name above (`TotCap` to `TotCapAdj`). The calculation here remains identical.